apollo-router/src/services/supergraph/snapshots/apollo_router__services__supergraph__tests__subscription_callback_schema_reload-3.snap

```diff
@@ -6,9 +6,9 @@ expression: stream.next_response().await.unwrap()
   "data": null,
   "errors": [
     {
-      "message": "subscription has been closed due to a schema reload",
+      "message": "subscription has been closed due to a router reload",
       "extensions": {
-        "code": "SUBSCRIPTION_SCHEMA_RELOAD"
+        "code": "SUBSCRIPTION_RELOAD"
```

Comment:
Will it confuse any clients receiving these errors that we have changed the code? Could anything be explicitly depending on the code being "SUBSCRIPTION_SCHEMA_RELOAD"?

Reply:
No I thought our clients were using this error code but not, they're just using a catch all on all errors and retry the connection if there's an error so we're good

Reply:
is this is better as a follow-up? I definitely think the `code` might be more sensibly named with the new token you've selected which more closely reflects that it might be the supergraph **or** the router configuration which have changed, which definitely doesn't matter to the client which.  This change just very much exceeds the scope of this PR which is about fixing hot-reloading.

Reply:
If we do change this, and I'm not convinced that we should because there may be clients we don't know about, then we need to clearly highlight it in the CHANGELOG entry and possibly in the docs.